.github/workflows/check.yml

```diff
@@ -26,14 +26,9 @@ jobs:
       - sre
     steps:
       - uses: actions/checkout@v2
-      - run: |
-          SKIP_WASM_BUILD=1 /home/runner/.cargo/bin/cargo check --all --benches
```

Comment:
I still think we need this, otherwise we're missing std check

Reply:
`runtime-benchmarks` and `std` are mutually exclusive features, personally of the opinion we don't need this #472 will run the actual runtime benchmarks if runtime changes are detected

Reply:
I've split the check for wasm benchmarks + std check up. As we have enough runnersm this shouldn't affect CI times and gives early feedback of incompatibility.